data/chains/hyper-evm.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "hyper-evm",
```

Comment:
As seen in the docs, HyperEVM is only the EVM component of the Hyperliquid blockchain, while the other part is HyperCore. Since there is only one EVM component, I'd prefer to use `hyperliquid`.